pyvista/plotting/_property.py

```diff
@@ -230,6 +237,9 @@ def __init__(
         self.line_width = line_width
         if culling is not None:
             self.culling = culling
+        if edge_opacity is None:
+            edge_opacity = self._theme.opacity
```

Comment:
I think we would generally want a different attribute in the theme for `edge_opacity` vs. `opacity`.

Reply:
Thanks!
```suggestion
            edge_opacity = self._theme.edge_opacity
```